code/modules/mob/living/living.dm

```diff
@@ -1721,6 +1721,10 @@ Thanks.
 		item.throw_at(target, item.throw_range*throw_mult, item.throw_speed*throw_mult)
 		return THREW_SOMETHING
 
+/mob/living/proc/handle_addictions()
```

Comment:
Shouldn't this be called at some point during Life()? Doesn't look like it is

Reply:
It's called now, in human/handle_chemicals_in_body